src/vm/comutilnative.cpp

```diff
@@ -1494,13 +1494,26 @@ FCIMPL5(VOID, Buffer::InternalBlockCopy, ArrayBase *src, int srcOffset, ArrayBas
 }
 FCIMPLEND
 
-void QCALLTYPE SpanNative::SpanClear(void *dst, size_t length)
+void QCALLTYPE MemoryNative::Clear(void *dst, size_t length)
 {
     QCALL_CONTRACT;
 
     memset(dst, 0, length);
 }
 
+bool QCALLTYPE MemoryNative::CopyWithReferences(void *dst, void *src, size_t byteCount)
+{
+    QCALL_CONTRACT;
```

Comment:
This cannot be QCall. You have to be in GC cooperative mode to do the copy of object references.